### DISCUSSION TITLE
5 m / 2 s  =  5m*s !

### DISCUSSION THREAD

**Discussion by LearningOpenGL** [hgrecco/pint#1785] [Q&A]
Hi,
I tried this:
velocity = ureg("5 m / 2 s")
it resulted in "2.5m*s" (i would have exspected "2.5m/s").
Ok, I can understand this is probably because the string is interpredted to be "5 * m / 2 * s". When I modify the string to "5 m / (2 s)" pint will give me the expected result "2.5m/s" so this behaviour might not be an issue.

My question:
Is there an option i can set in pint to always bind units to it's values? like "5 m / 2 s" ==> "(5 m) / (2 s)" ==> "2.5m/s"?

Thanks in advance,
Matthias

P.S.: this is my first post, i hope this is the correct place for my question.

**Comment by dalito** [comment]
Hi Matthias, there is no such option. As you found out you have to use brackets. The topic has come up before (for example https://github.com/hgrecco/pint/issues/1550).
